web/__test__/components/UserProfile.test.ts

```
@@ -0,0 +1,358 @@
+      vi.spyOn(console, 'debug').mockImplementation(() => {}),
+      vi.spyOn(console, 'info').mockImplementation(() => {}),
+      vi.spyOn(console, 'error').mockImplementation(() => {}),
+    ];
+
+    // Reset refs used by mocks
+    mockCopied.value = false;
+    mockIsSupported.value = true;
+
+    // Define mock Event classes
+    class MockEvent {
+      constructor(type: string, options?: unknown) {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)